src/NLog/Common/InternalLogger.cs

```diff
@@ -209,14 +214,14 @@ private static void Write([CanBeNull]Exception ex, LogLevel level, string messag
                 return;
             }
 
-            if (string.IsNullOrEmpty(LogFile) && !LogToConsole && !LogToConsoleError && LogWriter == null)
+            if (NoLogger())
```

Comment:
Although the method is also used once, It has a lot of statements. It's OK to create a new method, but please document the method (xmldocs)

Proposal for the name: "loggingEnabled" (and maybe a property is nicer)?